src/apps/explorer/components/OrderWidget/index.tsx

```diff
@@ -1,44 +1,31 @@
 import React, { useEffect, useState } from 'react'
-import styled from 'styled-components'
 import { useParams } from 'react-router'
 
 import { getOrder, RawOrder } from 'api/operator'
 
-import { OrderDetails } from './OrderDetails'
-
-// TODO: create a `View` component to abstract display elements away from logic/hooks
-const Wrapper = styled.div`
-  padding: 4rem 3rem;
-
-  > * {
-    margin-bottom: 2rem;
-  }
-`
+import { OrderWidgetView } from './view'
 
 export const OrderWidget: React.FC = () => {
+  // TODO: move order loading to a hook
   const [order, setOrder] = useState<RawOrder | null>(null)
+  const [isLoading, setIsLoading] = useState(false)
   const [error, setError] = useState('')
+  // TODO: load fills from a hook
 
   const { orderId } = useParams<{ orderId: string }>()
 
   useEffect(() => {
     // TODO: get network from a hook or something
+    setIsLoading(true)
     getOrder({ networkId: 4, orderId })
       .then((order) => {
         console.log('got order', order)
         setOrder(order)
         setError('')
       })
       .catch((e) => setError(e.message))
+      .finally(() => setIsLoading(false))
   }, [orderId])
 
-  return (
-    <Wrapper>
-      <h2>Order details</h2>
-      {order ? <OrderDetails order={order} /> : <p>Order not found</p>}
-      <h2>Order fills</h2>
-      <p>No fills</p>
-      {error && <p>{error}</p>}
-    </Wrapper>
-  )
+  return <OrderWidgetView order={order} isLoading={isLoading} error={error} />
```

Comment:
I vote against this naming convention. 
I prefer to call the dumb components without any suffix, they are just normal components.

First naming it `OrderWidgetView` u imply there's a 1<>1 relationship between OrderWidget and OrderWidgetView, which might be true here at least for now, although is not sth we always want to do. 

Another concern would be that the names get unnecesarilly complex. `MyCoolComponentWidgetViewStyledWrapperHoH` (disclaimer: some names in this film may or may not have been dramatized)

Reply:
What do you suggest?

Reply:
let's discuss [here](https://github.com/gnosis/gp-ui/issues/92)

Reply:
Good call opening the issue!